### ISSUE TITLE
Return address  

### ISSUE THREAD

**Issue by Jorgemgm** [thirtybees/thirtybees#536]
In the “Order Return”, pdf document, said “The package should be sent to the following address:” and the address is the same of “CONTACT DETAILS” of “Store Contacts” in “Preferences”.

Sometimes for ecommerce the return address isn’t the same of “Store Contacts”, I think it’s good idea create “Return address” under “Merchandise Returns” on menu “Orders”.

**Comment by doclucas** [comment]
Good idea, but not an issue, it's a feature request and should be posted here:
https://thirtybees.com/feature/?meta=_avfr_status&val=all

**Comment by doclucas** [comment]
@Traumflug if it stays open, I think it's best to add the "enhancement" flag to this one.

**Comment by doclucas** [comment]
Feature requests should be posted in the given link:
https://thirtybees.com/feature/?meta=_avfr_status&val=all

**Comment by Traumflug** [comment]
One can post them there, one can discuss features in the forum, one can open an issue for them. Each is fine.

**Comment by doclucas** [comment]
Alright, well noted.
